account_invoice_force_removal/models/account_move.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def unlink(self):
+        """Allow to remove moves related to invoices."""
+        for item in self:
+            if item.type in ["out_invoice", "out_refund", "in_invoice", "in_refund"]:
```

Comment:
Why limit this feature to only invoice types?

Reply:
Addon is limited to invoices moves because addon (`account_invoice_force_removal`) is related to invoices.

Reply:
You can change addon name if that is the problem 😉

Reply:
Ok, I'm going to change the name of the addon to `account_move_force_removal`, do you all agree? @pedrobaeza @sergio-teruel